src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -617,10 +617,9 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, OrgPersonMix
   .actions {
     position: absolute;
     right: 0;
-    margin-top: -0.5rem;
+    margin-top: -0.25rem;
 
     @extend .actions-width;
-    margin-top: -8px;
 
```

Comment:
Removed overall padding. margin-top at line 620 was being overridden by this (line 623). Reduced margin-top (line 620)  to align with the column on left. It is misaligned in DEV as well. This fixes the alignment for other filings (Change, Correction and Conversion). I tested for all the other filing types